.github/.release_notes_dependency_exclusion_list

```diff
@@ -0,0 +1,54 @@
+androidx.arch.core:core-testing
+androidx.compose.material3:material3
+androidx.compose.ui:ui
+androidx.compose.ui:ui-graphics
+androidx.compose.ui:ui-test-manifest
+androidx.compose.ui:ui-tooling
+androidx.compose.ui:ui-tooling-preview
+androidx.hilt:hilt-navigation-compose
+androidx.preference:preference-ktx
+androidx.test.espresso:espresso-contrib
+androidx.test.espresso:espresso-core
+androidx.test.espresso:espresso-intents
+androidx.test.ext:junit-ktx
+androidx.test.uiautomator:uiautomator
+androidx.test:rules
+app.cash.turbine:turbine
+com.adevinta.android:barista
+com.android.application
+com.android.tools.lint:lint
+com.android.tools.lint:lint-api
+com.android.tools.lint:lint-tests
+com.google.dagger.hilt.android
+com.google.dagger:hilt-android
+com.google.dagger:hilt-android-compiler
+com.google.dagger:hilt-android-testing
+com.google.dagger:hilt-compiler
+com.google.devtools.ksp
+com.google.pay.button:compose-pay-button
```

Comment:
Will this be part of the SDK with Google Pay improvements PR?

Reply:
Yes, so depending on which PR we move first, we should remove it either in this PR or in the Google Pay one

Reply:
I would say remove it from here regardless so that we don't forget it